packages/game/src/Emoji.ts

```diff
@@ -39,7 +40,7 @@ const defaultEmojis: {
 
 export function Emoji(name: Emojis): string {
   return `${
-    client.emojis.cache.find((emoji) => emoji.name === name) ??
+    // client.emojis.cache.find((emoji) => emoji.name === name) ??
```

Comment:
@brian-gates I need your help rethinking how Emojis are used with the client. As this is now treated like a library, there is no actual instantiation of a client in the library.